packages/lb-components/src/components/audioPlayer/index.tsx

```diff
@@ -808,6 +811,28 @@ export const AudioPlayer = ({
     setAudioUrl();
   }, [url]);
 
+  useEffect(() => {
+    getVisibleAreaRange();
+  }, [waveSize, duration]);
```

Comment:
Will it cause pool performance if change too quickly?

Reply:
Not changing quickly